### PR TITLE
Update RBAC documentation

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

**Review by tao-xinxiu — APPROVED** [review]
LGTM. Thank you.

**Review by aminelouati — APPROVED** [review]

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -2147,6 +2147,141 @@ It is possible to restore any previous commit to the HEAD position by clicking t
...
+
+* If multiple user-groups authorizations exist, without a username authorization, the resulting user's rights over the bucket will be the access type of the group authorization that have the highest priority.
+
+In the case where a user has multiple authorizations over an object, the resulting access type will be calculated as follows:
+
+* If the username-assigned authorization for the object exists, it is prioritized and its access type will be the user's resulting rights over the object.
+
+* If multiple user-groups authorizations exist for the object, without a username authorization, the resulting user's rights over the object will be the access type of the group authorization that have the highest priority.
+
+* If both username-assigned authorization and user-groups authorizations do not exist for the object, the resulting user's rights for the object will be the same as the user's resulting rights over the bucket that contains the object.
```
Comment (nebilbenmabrouk) [inline src/docs/user/ProActiveUserGuide.adoc:2283]: If both username-assigned authorization**s** and user-groups authorizations
(please use plural tense for homogeneity)

Comment (marounkoussaifi) [reply src/docs/user/ProActiveUserGuide.adoc:2283]: I'll update it. Thanks.

**Review by nebilbenmabrouk — APPROVED** [review]
LGTM

**Comment by tao-xinxiu** [comment]
retest this please

**Pull request merged**